README.md

```diff
@@ -82,7 +104,7 @@ Usage: iq-healthcheck [ARGS]...
   Example usage:
     
     # Run python script though docker container with all packages installed on it!
-    docker run -w /tmp --rm -i -v $PWD:/tmp broadinstitute/python-requests iq-scrape-conf.py -u "http://<iq-hostname>:<iq-port>"
+    docker run --rm -i -v $PWD:/data kehrhardt/iq-server-casc:latest iq-healthcheck -u "http://<iq-hostname>:<iq-port>"
```

Comment:
Please add the following to the arguments passed: -t /data/healthcheck/templates/<template-config>.json